build.gradle

```diff
@@ -23,6 +23,18 @@ plugins {
 apply plugin: 'com.automattic.android.fetchstyle'
 
 project.ext.buildGutenbergFromSource = project.properties.getOrDefault('wp.BUILD_GUTENBERG_FROM_SOURCE', false).toBoolean()
+def suppressJSBundle =  System.getenv('SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD').asBoolean()
+project.ext.buildGutenbergMobileJSBundle = !(suppressJSBundle || project.ext.buildGutenbergFromSource)
+
+if (project.ext.buildGutenbergFromSource) {
+    def nodeModulesDir = new File('libs/gutenberg-mobile/node_modules')
+    if (!nodeModulesDir.exists()) {
+        def message = "Attempting to build Gutenberg from source without a libs/gutenberg-mobile/node_modules directory. \
```

Comment:
definitely appreciate this message as a reminder! 🙏

Reply:
Nothing like tripping over something multiple times yourself to make you think, "Hmmm, maybe I should give a helpful message here." 😆